insights/specs/default.py

```diff
@@ -61,6 +61,9 @@ def inner(idx=None):
     return inner
 
 
+format_rpm = _make_rpm_formatter()
```

Comment:
@danzatt good catch on this, I'm not sure how this slipped passed my diff output in the previous commit. But the issue is actually with the inner function in _make_rpm_formatter. If you could instead remove the inner function since it's not actually used at all for it's purpose that I can tell, and instead replace it with `return "\{" + ",".join(fmt) + "\}\n"`.

Reply:
Sorry I missed your comment @ryan-blakley and just released @danzatt's version in a rush. Now I raised anther PR according to Ryan's comment.